symengine/integer.h

```diff
@@ -208,6 +208,20 @@ inline Integer::Integer(integer_class i_) : i{std::move(i_)}
 {
 }
 
+// returns the integer if `x` is a positive integer else
+// returns `-1` if negative integer, `0` if not an integer 
+inline int positive_integer(const RCP<const Basic> &x)
+{
+    if (is_a<const Integer>(*x)) {
+        int tmp = rcp_static_cast<const Integer>(x)->as_int();
```

Comment:
No, but not only do I need the `sign` I also need the value (as an `int`) if it's positive. Also, the reason I made this function is that I could directly call it on a `Basic` and get the job done, instead of `is_a` and `rcp_cast` every time. Where do you want me to place it?